app/ide-desktop/lib/dashboard/craco.config.ts

```diff
@@ -0,0 +1,52 @@
+/** @file Configuration file for the `craco` package.
+ *
+ * `craco` is used in place of the default `react-scripts` package provided by `create-react-app`.
+ * `craco` is a wrapper around `react-scripts` that allows us to customize the build process. We use
+ * it to disable the `ModuleScopePlugin`. The `ModuleScopePlugin` prevents importing modules outside
+ * of the `src` directory. This package imports files from sibling packages in the monorepo, so the
```

Comment:
Currently this is used to import constants from `shared.ts` in the `content` package. `shared.ts` is part of the content build scripts. So it is not exported from the content package. Hence we we import just the file, by relative path.

Exporting the constants we need from content would not work either, since content+dashboard would have circular dependencies on each other.

I could introduce a standalone package for shared constants for use by content and dashboard.

Reply:
@indiv0 how long do you estimate "I could introduce a standalone package for shared constants for use by content and dashboard."?

Reply:
This has been done already.

Reply:
@indiv0 ok thanks